omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "omnibus", git: "https://github.com/chef/omnibus.git"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git"
+gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git", branch: "delivery_cli"
```

Comment:
That branch needs to be merged before we merge this - https://github.com/chef/omnibus-software/pull/655